src/Nancy.Testing/Browser.cs

```diff
@@ -219,7 +219,9 @@ public BrowserResponse HandleRequest(string method, Url url, Action<BrowserConte
             var request =
                 CreateRequest(method, url, browserContextValues);
 
-            var response = new BrowserResponse(this.engine.HandleRequest(request), this, (BrowserContext)browserContextValues);
+            var context = await this.engine.HandleRequest(request);

```

Comment:
**Question**: anywhere we do an await within Nancy, should we also `.ConfigureAwait(false)`. My understanding is that all libraries should be doing that.